clientv3/op.go

```diff
@@ -289,6 +289,7 @@ func withTop(target SortTarget, order SortOrder) []OpOption {
 
 // WithProgressNotify makes watch server send periodic progress updates.
```

Comment:
send periodic progress updates every 10 minutes even if there is no actual events.